docs/streams/developer-guide/write-streams.html

```diff
@@ -38,13 +38,13 @@
           <li><a class="reference internal" href="#libraries-and-maven-artifacts" id="id1">Libraries and Maven artifacts</a></li>
           <li><a class="reference internal" href="#using-kafka-streams-within-your-application-code" id="id2">Using Kafka Streams within your application code</a></li>
       </ul>
-    <p>Any Java application that makes use of the Kafka Streams library is considered a Kafka Streams application.
+    <p>Any Java or Scala application that makes use of the Kafka Streams library is considered a Kafka Streams application.
       The computational logic of a Kafka Streams application is defined as a <a class="reference internal" href="../concepts.html#streams-concepts"><span class="std std-ref">processor topology</span></a>,
       which is a graph of stream processors (nodes) and streams (edges).</p>
     <p>You can define the processor topology with the Kafka Streams APIs:</p>
     <dl class="docutils">
       <dt><a class="reference internal" href="dsl-api.html#streams-developer-guide-dsl"><span class="std std-ref">Kafka Streams DSL</span></a></dt>
-      <dd>A high-level API that provides provides the most common data transformation operations such as <code class="docutils literal"><span class="pre">map</span></code>, <code class="docutils literal"><span class="pre">filter</span></code>, <code class="docutils literal"><span class="pre">join</span></code>, and <code class="docutils literal"><span class="pre">aggregations</span></code> out of the box. The DSL is the recommended starting point for developers new to Kafka Streams, and should cover many use cases and stream processing needs.</dd>
+        <dd>A high-level API that provides provides the most common data transformation operations such as <code class="docutils literal"><span class="pre">map</span></code>, <code class="docutils literal"><span class="pre">filter</span></code>, <code class="docutils literal"><span class="pre">join</span></code>, and <code class="docutils literal"><span class="pre">aggregations</span></code> out of the box. The DSL is the recommended starting point for developers new to Kafka Streams, and should cover many use cases and stream processing needs.  If you're writing a Scala application you can use the <a href="dsl-api.html#streams-developer-guide-scala"><span class="std std-ref">Scala Kafka Streams DSL</span> library which removes the Java/Scala integraiton boilerplate when working with the Java DSL.</a></dd>
```

Comment:
This is not introduced in this PR: duplicated `provides`.

Reply:
I fixed the typos in this line, but I'm not sure what you mean by it not being introduced in this PR.  This line is to indicate the presence of the Kafka Streams DSL for Scala library.

Reply:
I meant the duplicated `provides` exist before this PR, so it is not a regression introduced from this PR.